arbeitszeit_flask/templates/macros/draft_form.html

```diff
@@ -39,18 +36,21 @@
 </div>
 <div class="block py-2"></div>
 <div class="box has-background-info-light">
-  <div class="has-text-right has-text-info-dark js-modal-trigger" data-target="id_unit_amount">
-    <span class="icon"><i class="fa-solid fa-circle-question"></i></span>
-  </div>
   <div class="field">
+    <div class="has-text-right has-text-info-dark">
+      {{ info_tooltip(id="id_unit", text=gettext("This value is used for calculation purposes only, i.e. you do not have to adhere to any rules here. For example, if you plan to produce beverages, you are free to choose if your smallest delivery unit is one can or one six-pack of cans."))}}
+    </div>
     <label class="label">{{ gettext("Smallest delivery unit")}}</label>
     <div class="control">
       {{ form.prd_unit(class_="input is-large") }}
-      <p class="help">{{ gettext("E.g. 1 package of 100 pieces, 1 kilogram, 1 language lesson of 1.5 hour, etc.")}}</p>
+      <p class="help">{{ gettext("E.g. 1 package, 1 kilogram, 20 bottles, 1 language lesson, etc.")}}</p>
     </div>
   </div>
   <div class="block"></div>
   <div class="field">
+    <div class="has-text-right has-text-info-dark">
+      {{ info_tooltip(id="id_amount", text=gettext("This value is assumed to be in the same unit as Smallest delivery unit. By dividing the sum of your production costs through this amount, the app calculates how much labour time is required to produce one piece of your product."))}}
```

Comment:
"Smallest" -> "smallest"
"divide BY"

What about changing the first sentence?

'How many units, as described in "smallest delivery unit", will you produce? By dividing the sum of your production costs by this amount, the app calculates how much labour time is required to produce one piece of your product.'

Reply:
done, changed first sentence to "How many of your \"smallest delivery unit\" will you produce in total?"